pages.ta/common/cksum.md

```diff
@@ -0,0 +1,9 @@
+# cksum
+
+> கோப்பின் CRC சரிகாண்தொகையைக் கணித்து அதில் எத்தனை எண்ணிருமிகளுள்ளன என்றெண்ணு.
+> குறிப்பு: பழைய Unix கணினிகளில் CRC கணிமுறை மாறலாம்.
+> மேலும் விவரத்திற்கு: <https://www.gnu.org/software/coreutils/cksum>.
+
+- கோப்பின் 32-இருமி சரிகாண்தொகையையும் எண்ணிருமி அலகில் கோப்பளவையும் பெயரையும் காட்டு:
+
+`cksum {{கோப்பு}}`
```

Comment:
Why you didn't translate the `path/to/file` placeholder as-is?

Reply:
Answered below:

> Tamil is an agglutinative language. This means that what are phrases in
English often get translated to one giant word in Tamil. These giant
words can be quite a mouthful and hard to read. For example,
"path/to/file" translates to "கோப்புப்பாதை", an agglutination of two
words. I feel that just "கோப்பு" (meaning file) sounds clearer and more
concise.